integration/container-injection.c

```diff
@@ -0,0 +1,12 @@
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+const char *message = "Injected!\n";
+
+__attribute__((constructor))
+static void init()
+{
+    write(1, message, strlen(message));
+    exit(0);
```

Comment:
I think it's probably better to pipe the output of the process through `grep -F "Injected"`, but it requires rethinking how to find the PID. I think I ought to do that.

Although an advantage is that it ends the test as soon as the injection is done.